base_sparse_field_list_support/__manifest__.py

```diff
@@ -1,13 +1,12 @@
-# -*- coding: utf-8 -*-
 {
     "name": "Base Sparse Field List Support",
     "summary": "add list support to convert_to_cache()",
-    "version": "10.0.1.0.0",
+    "version": "12.0.1.0.0",
     "category": "Technical Settings",
     "website": "www.akretion.com",
     "author": "Akretion",
     "license": "AGPL-3",
     "application": False,
-    "installable": False,
+    "installable": True,
     "depends": ["base"],
```

Comment:
@rousseldenis you must depends on [base_sparse_field](https://github.com/odoo/odoo/tree/12.0/addons/base_sparse_field)

Reply:
Ok, integrated in Odoo now.